packages/runtime/src/useCases/transport/messages/SendMessage.ts

```diff
@@ -133,6 +136,27 @@ export class SendMessageUseCase extends UseCase<SendMessageRequest, MessageDTO>
         return;
     }
 
+    private validateMailRecipients(content: Serializable, recipients: string[]): ApplicationError | undefined {
```

Comment:
I would rename the method as it gives the impression that the Mail recipients are passed as input parameters when in fact the Message recipients are passed.

Reply:
```suggestion
    private validateMailRecipients(mail: Serializable, messageRecipients: string[]): ApplicationError | undefined {
```
@britsta I think the method name is fine, the argument wording is not.

Reply:
Also fine for me @jkoenig134